src/codeflare_sdk/templates/base-template.yaml

```diff
@@ -289,38 +289,46 @@ spec:
                   emptyDir: {}
     - replicas: 1
       generictemplate:
-        kind: Route
-        apiVersion: route.openshift.io/v1
+        apiVersion: networking.k8s.io/v1
+        kind: Ingress
         metadata:
-          name: ray-dashboard-deployment-name
+          name: ray-dashboard-raytest
           namespace: default
-          labels:
-            # allows me to return name of service that Ray operator creates
-            odh-ray-cluster-service: deployment-name-head-svc
         spec:
-          to:
-            kind: Service
-            name: deployment-name-head-svc
-          port:
-            targetPort: dashboard
+          rules:
+            - host: placeholder
+              http:
+                paths:
+                  - path: /
+                    pathType: Prefix
+                    backend:
+                      service:
+                        name: raytest-head-svc
+                        port:
+                          number: 8265
     - replicas: 1
       generictemplate:
-        apiVersion: route.openshift.io/v1
-        kind: Route
+        apiVersion: networking.k8s.io/v1
+        kind: Ingress
         metadata:
           name: rayclient-deployment-name
           namespace: default
           labels:
-            # allows me to return name of service that Ray operator creates
             odh-ray-cluster-service: deployment-name-head-svc
         spec:
-          port:
-            targetPort: client
-          tls:
-            termination: passthrough
-          to:
-            kind: Service
-            name: deployment-name-head-svc
+          rules:
+            - host: "localhost"
```

Comment:
I think this should be a FQDN i.e. like rayclient-raytest.<ingress domain>. See if you could utilize the [_get_ingress_domain](https://github.com/project-codeflare/codeflare-sdk/blob/fb34ba4fdff6b586db990461e694a1b94e3c185a/src/codeflare_sdk/cluster/cluster.py#L434)() method to get the ingress domain.